### PR TITLE
USWDS-Site: Add clarity to Phase 2: compile instructions

### LINKED ISSUE #2061: USWDS-Site - Confusion over setting up Sass entry point
## Summary
We should evaluate the guidance for creating a Sass entry point and configuring USWDS settings to make sure it makes sense when not using `uswds-compile`.

There are some opportunities to add clarity: 
The [Settings page](https://designsystem.digital.gov/documentation/settings/) refers to the settings configuration file as `'uswds-theme'`, but the Getting Started for Developers guide refers to it as `uswds-settings.scss`. This can lead users to think they are two separate files, which can be confusing. 

There is also an opportunity to add clarity about which files will be provided by USWDS and which will need to be created by the user. 

## More details
This issue was originally flagged in [this Slack thread](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1681831091624819).

### Steps to reproduce the bug

_No response_

### Expected Behavior

_No response_

### Screenshots

_No response_

### System setup

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Updated the guidance in [Phase 2: Compile](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/) to add clarity by:
1. Re-organizing content to more clearly identify which instructions are for `uswds-compile` users and which are for users who compile Sass a different way
1. Updating the file names in the code demos to match what `uswds-compile` creates
1. Adding copy to clarify which files are project files and which are USWDS files

⚠️ The changelog date will need to be updated before merge.

## Related issue

Closes #2061

## Preview link

Preview link: [Phase 2: Compile](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-update-compile-instructions/documentation/getting-started/developers/phase-two-compile/)

## Problem statement

There is an opportunity to add clarity about which files will be provided by USWDS and which will need to be created by the user. 

This issue was originally flagged in [this Slack thread](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1681831091624819).

## Testing and review
- Confirm that there are no spelling or grammatical errors
- Confirm the file names in the code samples matches what is generated by `uswds-compile` 
- Confirm that the content order makes sense

### CODE REVIEW

File: pages/documentation/getting-started-developers/phase-two.md [hunk 1]
```diff
@@ -34,25 +34,26 @@ U.S. Web Design System source code is written in Sass, a powerful stylesheet lan
...
 A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase. Often, a project's Sass entry point is named something like `index.scss` or `styles.scss`.
 
-### Setting up USWDS in your Sass entry point
-If your project does not yet have a Sass entry point, create a file called `index.scss` or `styles.scss`.
+### Creating a Sass entry point
+If your project does not yet have a Sass entry point, create a file called `styles.scss`. This file will typically live at the root of your project's Sass directory.
 
- Your project’s Sass entry point is a simple file that will only need to do the following three tasks:
+### Loading USWDS in your Sass entry point
+Your project’s Sass entry point is a simple file that will need to do the following three tasks:
```
Comment (bonnieAcameron) [inline pages/documentation/getting-started-developers/phase-two.md:54]:
```suggestion
Your project’s Sass entry point is a simple file that will need to:
```

**Review by bonnieAcameron — CHANGES_REQUESTED** [review]
Hi @amyleadem ! These changes are all so great. I have a small suggestion about paring down the wordiness on a list of steps, but other than that it looks fantastic.

**Comment by bonnieAcameron** [comment]
As an aside @amyleadem , do you think I could pick your brain about when/how/why you use `code styling` on your documentation? I think it's really useful and I think it would be smart to add it to a USWDS style guide or operationalize it in some way.

**Review by mahoneycm — COMMENTED** [review]
Checked over the instructions and they're definitely improved in my opinion! I appreciate you taking the time to do this.

- [x] Content is free of spelling and grammar errors
- [x] File names match the output files from `uswds-compile`
- [x] Content order is logical

I added one small comment about potentially adding a note on init instructions but looks good either way!

File: pages/documentation/getting-started-developers/phase-two.md (hunk 1 above)
Comment (amyleadem) [reply pages/documentation/getting-started-developers/phase-two.md]: Updated in https://github.com/uswds/uswds-site/pull/2123/commits/fd33a9d8fdc5df7d4ed8b06bc68c5b8152fcdf4b

File: pages/documentation/getting-started-developers/phase-two.md
[the remark this answers is not in the document]
Comment (amyleadem) [reply pages/documentation/getting-started-developers/phase-two.md:1]: Good suggestion. I've added an update in [0dc681c](https://github.com/uswds/uswds-site/pull/2123/commits/0dc681c9977d4fe776cf39489d6c0fd5d3c37293). Let me know if you think that does the job.

**Comment by amyleadem** [comment]
@bonnieAcameron @mahoneycm 
I've added some updates according to your suggestions. Will you take another look?

Also, @bonnieAcameron I would be happy to share whatever I know about code documentation. Let's find some time to chat.

**Review by mahoneycm — APPROVED** [review]
Looks good! I think that note along with the sample `gulp.js` file should be more than enough to keep people on the right track! 👍

**Review by bonnieAcameron — APPROVED** [review]
Looks great! The changes to this page make it so much more readable.

File: pages/documentation/getting-started-developers/phase-two.md [hunk 2]
```diff
@@ -61,23 +62,23 @@ For your entry point to complete these tasks, you will need to add the following
...
-@forward "uswds-settings.scss";
+// 1. Load your project's USWDS settings configuration
+@forward "uswds-theme.scss";
 
 // 2. Load USWDS source code
 @forward "./path/to/source/uswds";
 
 // 3. Load your project's custom Sass
-@forward "project-custom.scss";
+@forward "uswds-theme-custom-styles.scss";
```
Comment (mejiaj) [inline pages/documentation/getting-started-developers/phase-two.md:72]: The previous `project-custom.scss` seems clearer that they are project specific styles.

File: pages/documentation/getting-started-developers/phase-two.md [hunk 3]
```diff
@@ -34,25 +34,26 @@ U.S. Web Design System source code is written in Sass, a powerful stylesheet lan
 
 ## Introducing uswds-compile
 
-We’ve developed `uswds-compile`, a tool [hosted on GitHub](https://github.com/uswds/uswds-compile), to help teams compile USWDS Sass files. This tool provides simple Gulp 4 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
+If you don't yet have Sass compilation set up on your project, we recommend using `uswds-compile`, a USWDS tool [hosted on GitHub](https://github.com/uswds/uswds-compile), as a quick way to get up and running with Sass compilation for USWDS.
```
Comment (mejiaj) [inline pages/documentation/getting-started-developers/phase-two.md:37]: ## Optional

Shortening could help with clarity.

```suggestion
We recommend using `uswds-compile`, a USWDS tool [hosted on GitHub](https://github.com/uswds/uswds-compile), as a quick way to get up and running with Sass compilation for USWDS.
```

File: pages/documentation/getting-started-developers/phase-two.md [hunk 4]
```diff
@@ -34,25 +34,26 @@ U.S. Web Design System source code is written in Sass, a powerful stylesheet lan
...
 
-## About Sass entry points
+### About Sass entry points
 
 A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase. Often, a project's Sass entry point is named something like `index.scss` or `styles.scss`.
 
-### Setting up USWDS in your Sass entry point
-If your project does not yet have a Sass entry point, create a file called `index.scss` or `styles.scss`.
+### Creating a Sass entry point
+If your project does not yet have a Sass entry point, create a file called `styles.scss`. This file will typically live at the root of your project's Sass directory.
```
Comment (mejiaj) [inline pages/documentation/getting-started-developers/phase-two.md:51]: The reference to `index.scss` or `styles.scss` seemed useful.

**Review by mejiaj — CHANGES_REQUESTED** [review]

File: pages/documentation/getting-started-developers/phase-two.md (hunk 4 above)
Comment (amyleadem) [reply pages/documentation/getting-started-developers/phase-two.md]: Restored `index.scss` text in https://github.com/uswds/uswds-site/pull/2123/commits/4661914f8f766e16e80bc1b5f1f648aad2d1783b

File: pages/documentation/getting-started-developers/phase-two.md (hunk 2 above)
Comment (amyleadem) [reply pages/documentation/getting-started-developers/phase-two.md]: That makes sense. I updated the name to `project-styles.scss` in [4661914](https://github.com/uswds/uswds-site/pull/2123/commits/4661914f8f766e16e80bc1b5f1f648aad2d1783b) to be more descriptive. Does that work for you?

**Comment by amyleadem** [comment]
@mejiaj I've made your requested changes. Let me know if you have any questions.

**Review by mejiaj — COMMENTED** [review]
LGTM, content makes sense and is accurate. Should we add a changelog entry?

I tested:
- [x] Content makes sense & is accurate.
- [x] Zero broken links.
- [x] Zero axe errors.

**Comment by amyleadem** [comment]
@mejiaj A changelog entry is a good idea. Thanks! I've added an entry in [3854033](https://github.com/uswds/uswds-site/pull/2123/commits/38540337d04979240da983357a654c27cf486c0b). Let me know what you think.

File: _data/changelogs/docs-getting-started-devs-phase-2.yml [hunk 5]
```diff
@@ -2,6 +2,11 @@ title: "Phase 2: Compile"
 type: documentation
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```
Comment (thisisdano) [inline _data/changelogs/docs-getting-started-devs-phase-2.yml:5]:
```suggestion
  - date: 2023-07-28
```

**Review by thisisdano — APPROVED** [review]

**Pull request merged**